pytximport/core/_tximport.py

```diff
@@ -553,7 +576,17 @@ def tximport(
 
         log(25, f"Saving the gene-level expression to: {output_path}.")
 
-        if output_format == "h5ad":
+        if output_format.lower() == "summarizedexperiment":
+            from summarizedexperiment import SummarizedExperiment
+            from dolomite_base import save_object
+            import dolomite_se  # noqa: F401
```

Comment:
Why is this module imported but F401 ignored? If it is not used, it should not be imported

Reply:
its used internally by `dolomite_base`